Automattic/newspack-theme
# feat: add Threads logo to the social links menu

### All Submissions:


### Changes proposed in this Pull Request:

This PR adds Threads to the icons available in the social links menu.

See 1200550061930446-as-1204990480671811

### How to test the changes in this Pull Request:

1. Apply the PR.
2. Set up a social links menu, and add a link to a Threads account (eg. https://www.threads.net/@nytimes)
3. View on the front-end; confirm that the link is using the Threads logo.

![image](https://github.com/Automattic/newspack-theme/assets/177561/51247bc5-bc2b-4494-a829-bce76a01c2e5)

### Other information:
